airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -2008,7 +2008,7 @@
   documentationUrl: https://docs.airbyte.com/integrations/sources/twitter
   icon: twitter.svg
   sourceType: api
-  releaseStage: alpha
+  releaseStage: beta
```

Comment:
Please not to update release stage, before you get approve from PM side in check list